src/java/com/google/cloud/bigquery/dwhassessment/hooks/logger/EventRecordConstructor.java

```diff
@@ -15,8 +15,8 @@
  */
 package com.google.cloud.bigquery.dwhassessment.hooks.logger;
 
-import static com.google.cloud.bigquery.dwhassessment.hooks.logger.LoggerVarsConfig.MR_QUEUE_NAME;
-import static com.google.cloud.bigquery.dwhassessment.hooks.logger.LoggerVarsConfig.TEZ_QUEUE_NAME;
+import static com.google.cloud.bigquery.dwhassessment.hooks.utils.LoggerVarsConfig.MR_QUEUE_NAME;
```

Comment:
I don't feel `utils` is a good package name for enums and `LoggerVarsConfig`. For the latter, I believe `config` suits more. Not sure about enums – I would maybe just leave them as is, but I probably miss some context here.